helm_deploy/dps-data-compliance/templates/housekeeping-cronjob.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1 CronJob
+kind: CronJob
+metadata:
+  name: {{ include "app.fullname" . }}-queue-housekeeping-cronjob
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  schedule: "0 9 * * 1-5"
```

Comment:
Monday to Friday at 9am Each day. This will not need to be as frequent as other services.